bin/LAUNCH.sh

```diff
@@ -497,7 +497,7 @@ function reset_password() {
 
 function launch_all {
     checking_configuration;
-    update;
+    #update;
```

Comment:
You shouldn't remove the updater (enabled by default).
You can edit ``configs/update.cfg`` if you want to disable it.